JeMPI_Apps/JeMPI_BackupRestoreAPI/src/main/java/org/jembi/jempi/backuprestoreapi/PsqlAuditTrail.java

```diff
@@ -0,0 +1,100 @@
+package org.jembi.jempi.backuprestoreapi;
```

Comment:
@sushantpatil1214 I dont think we need this file too

Reply:
Its packge defination.